src/control/lib/support/log.go

```diff
@@ -61,10 +62,11 @@ type LogTypeSubCmd struct {
 }
 
 const (
-	MMDD        = "1-2"
-	HHMMSS      = "15:04:05"
-	MMDDHHMMSS  = "1/2-15:04:05"
-	MMDD_HHMMSS = "1/2 15:04:05"
+	MMDD            = "1-2"
+	HHMMSS          = "15:4:5"
+	MMDDHHMMSS      = "1/2-15:4:5"
+	MMDD_HHMMSS     = "1/2 15:4:5"
+	YYYYMMDD_HHMMSS = "2006/1/2 15:4:5"
```

Comment:
I find this mapping strange as the values don't seem to match the format described by the name of the constant

Reply:
I found the issue when constant is having multiple digits and we pass the single digit. But if constant has the single digit and we pass the multiple digit works as expected. So technically the format is single digit but it can accept the multiple digit.